app/components/podium_component/podium_component.html.erb

```diff
@@ -1,15 +1,28 @@
 <div<%= " id=#{id}" if id %> class="podium-component<%= classes %>">
-  <% if podium.includes_school? %>
-    <h4 class="text-center">
-      <%= t('components.podium.position_html', position_ordinal: podium.school_position.ordinal) %>.
-    </h4>
+  <% if podium.includes_school? && podium.school_has_points? %>
+    <% if EnergySparks::FeatureFlags.active?(:activities_2023) %>
+      <h4 class="text-center"><%= t('components.podium.full_position_html', position_ordinal: podium.school_position.ordinal, scoreboard: podium.scoreboard.name, scoreboard_path: scoreboard_path(podium.scoreboard)) %>.</h4>
+    <% else %>
+      <h4 class="text-center"><%= t('components.podium.position_html', position_ordinal: podium.school_position.ordinal) %>.</h4>
+    <% end %>
+  <% else %>
+    <h4 class="text-center"><%= t('components.podium.no_points_this_year') %></h4>
+
+    <p class="text-center"><%= t('components.podium.complete_an_activity') %>.
```

Comment:
Suggestion: link the "complete an activity" text to the school recommendations page, so there's a direct link to that page from the prompt.

Reply:
Good suggestion, thank you.